companies/tasks/report_processing.py

```
@@ -0,0 +1,231 @@
+            "submitted_at": str(report.submitted_at),
+        }
+    except Report.DoesNotExist:
+        logger.error(f"Report not found: {report_id}")
+        return None
+    except Exception as e:
+        logger.error(f"보고서 본문 추출 오류: {report_id} - {e}")
+        raise self.retry(exc=e, countdown=60)
+
+
+@shared_task(bind=True, max_retries=3)
+def refine_report_content_task(self, data: dict[str, Any]) -> dict[str, Any] | None:
```

Ruff (0.14.11)

Unused function argument: `self`

(ARG001)